docs/source/serving/update.rst

```diff
@@ -0,0 +1,132 @@
+.. _serve-update:
+
+Update
+===========
+
+SkyServe supports update for your services. Use ``sky serve update`` to update an existing service:
+
+.. code-block:: console
+
+    $ sky serve update service-name new_service.yaml
+
+SkyServe will launch new replicas described by ``new_service.yaml``. When the number of new replicas reaches the minimum number of replicas (``min_replicas``) required for the service, SkyServe will scale down old replicas to save cost. SkyServe allows users to update ``replica_policy`` parameters, such as ``target_qps_per_replica``. SkyServe also allows users to update ``resources`` parameters, such as ``cpu`` and ``memory``, so that new replicas can be launched on VMs of different types.  
```

Comment:
```suggestion
SkyServe will launch new replicas described by ``new_service.yaml``. When the number of new replicas reaches the minimum number of replicas (``min_replicas``) required for the service, SkyServe will scale down old replicas to save cost. During the whole process, the service is still accessible for users. SkyServe allows users to update ``replica_policy`` parameters, such as ``target_qps_per_replica``. SkyServe also allows users to update ``resources`` parameters, such as ``cpu`` and ``memory``, so that new replicas can be launched on VMs of different types.  
```